README.md

```diff
@@ -47,48 +47,35 @@ $ bin/cqlsh
 To create keyspaces in a single node cluster:
 
 ```cql
-CREATE KEYSPACE IF NOT EXISTS <KEYSPACE_NAME_NETWORK_STORE> WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
-CREATE KEYSPACE IF NOT EXISTS <KEYSPACE_NAME_GEO_DATA> WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1};
+CREATE KEYSPACE IF NOT EXISTS iidm WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
+CREATE KEYSPACE IF NOT EXISTS geo_data WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1};
```

Comment:
Why did you replace `<KEYSPACE_NAME_NETWORK_STORE>` and `<KEYSPACE_NAME_GEO_DATA>` with their default values (`iidm` and `geo_data`) ? Since the keyspaces names can be modified in config files, I would have kept <KEYSPACE_NAME_NETWORK_STORE> and <KEYSPACE_NAME_GEO_DATA> and modified following lines (replace `cgmes_boundary` by `<KEYSPACE_CGMES_BOUNDARY>` and so on), maybe specifying their default value in a comment line. For example:
```cql
CREATE KEYSPACE IF NOT EXISTS <KEYSPACE_NAME_NETWORK_STORE> WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
// default value for <KEYSPACE_NAME_NETWORK_STORE> is "iidm"
```
This part of doc is confusing since it is not clear that user can use custom keyspaces names.

Reply:
Both solutions are good. There were just some commands with those <KEYSPACE_NAME_XXX>, and others with default values, with a parameter to change names for the first and no possible way to change the name for last.
I am in favour of homogenizing the operation, that's all.

Reply:
Actually I prefer a command that you can Copy/Paste simply and works directly, even if it's with default names/values.
It doesn't means you can't choose your own custom names, but a note after commands should give instructions to do that.

Reply:
Ok as discussed this morning, I put commands with <XXX> AND the commands with default values. Tell me if it's ok for you ?